package.json

```diff
@@ -25,22 +27,25 @@
     "prepare": "patch-package",
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx,js,jsx}'",
     "prettier-write": "yarn run prettier --write",
-    "publish-assets:local": "yarn assets && yarn publish-assets",
     "publish-assets": "node scripts/uploadToS3.js",
+    "publish-assets:local": "yarn clean && yarn build:assets && yarn publish-assets",
     "relay": "relay-compiler",
     "sentry": "scripts/sentry.sh",
-    "start:dev": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src",
-    "start:prod:local": "NODE_ENV=production node --inspect --max_old_space_size=3072 -r dotenv/config ./server.dist.js",
-    "start:prod": "yarn assets && yarn build:server && NODE_ENV=production yarn start",
     "start": "scripts/start.sh",
+    "start:dev": "concurrently 'yarn start:dev:relay' 'yarn start:dev:server'",
+    "start:dev:server": "NODE_ENV=development BUILD_CLIENT=true node --inspect --max_old_space_size=3072 -r dotenv/config ./src/index.dev.js",
+    "start:dev:relay": "NODE_ENV=development yarn relay --watch",
+    "start:prod": "yarn build:assets && yarn build:server && NODE_ENV=production yarn start",
+    "start:prod:local": "NODE_ENV=production node --inspect --max_old_space_size=3072 -r dotenv/config ./server.dist.js",
```

Comment:
What does "local" mean in this case? Perhaps `debug` might communicate the intent better since we're attaching the inspector?

Reply:
Thoughts on using `start:prod:debug`?

Reply:
Would it be possible to have a linear conversation? I'm struggling with the constant jumping between threads to discuss related topics.

Reply:
I would suggest that we migrate to standard npm/yarn naming conventions long term.

* yarn start
* yarn dev
* yarn test

and then add one-offs as needed using `:` is a visual way to express intent, that a task belongs to a related topic, and/or is a subtask of an overall task.

`yarn jest` and `yarn mocha` appear to me as being purely historical in meaning due to how the CircleCI orbs were previously configured.

I feel like `debug` is a practical change for `local`. If you weren't aware, for very simple changes like this Github has a handy feature built-in that allows you to suggest the change so that it may be committed directly from the PR.

My issues with the scripts has little to do with bash, I ❤️ bash and would write force in it if I could. My issue is more straightforward. You cannot execute the scripts from the command line:

```sh
# errors
./scripts/acceptance.sh src/test/acceptance/*.js
```

Remember the goal in the PR is to enable the next steps in development for novo without any developer needing to change their current workflow. If you would like to consolidate the commands and scripts to clarify naming and usage, I welcome any outside help.

Reply:
This makes sense 👍